outputs.tf

```diff
@@ -17,3 +17,9 @@ output "validation_id" {
   value       = join("", aws_acm_certificate_validation.default.*.id)
   description = "The ID of the certificate validation"
 }
+
+output "validation_arn" {
+  value       = join("", aws_acm_certificate_validation.default.*.certificate_arn)
```

Comment:
It's just the ID, the listener needs an arn

Reply:
yes, but just the arn of the certificate, but I need the arn of the certificate_validation resource to avoid errors while applying the ALB

Reply:
Maybe I'm missing something, but from the documentaion of `aws_acm_certificate `resource:
`It deals with requesting certificates and managing their attributes and life-cycle. This resource does not deal with validation of a certificate but can provide inputs for other resources implementing the validation. It does not wait for a certificate to be issued. Use a aws_acm_certificate_validation resource for this.`

That's the reason I want to add the validation arn, because the `aws_acm_certificate` resource doesn't wait for the certificate to be issued, the arn of the `acm_certificate_validation ` do.
When I'm adding the `aws_acm_certificate` arn to the ALB listener the first apply of the Terraform stack will fail until the certificate is issued, with the ARN of `acm_certificate_validation ` everything working fine even in the first apply.
Sorry if I have not explained this sufficiently.